support_bundle.sh

```diff
@@ -6,6 +6,11 @@ else
   namespace="$1"
 fi
 
+if [ -z "$namespace" ]; then
```

Comment:
in the name of consistency
```suggestion
if [ -z "${namespace}" ]; then
```